engine/spec/components/footer_spec.rb

```diff
@@ -3,9 +3,7 @@
 RSpec.describe CitizensAdviceComponents::Footer, type: :component do
   subject(:component) do
     render_inline(CitizensAdviceComponents::Footer.new(feedback_url: feedback_url)) do |c|
-      columns.each do |column|
-        c.column(column)
-      end
+      c.columns(columns)
```

Comment:
We can also do:

```
columns.each do |column|
   c.column(title: column[:title, links: column: column[:links])
end
```

But using the plural form of the method for passing an array is more conventional for view component slots.